### ISSUE TITLE
Namespacing

### ISSUE THREAD

**Issue by pomegranited** [zyegfryed/django-statici18n#40]
Hello @zyegfryed et al!

We've got a use case for being able to generate Javascript namespaced `gettext` commands, essentially adding a wrapper for your generated JS, like this:

```javascript
(function(global){
    var MyLittleApp = {
      init: function() { 
        {{generated djangojs.js goes here}}
      }
    };
    global.MyLittleApp = MyLittleApp;
}(this);

...

translated_text = MyLittleApp.gettext("text from my little app");
```

Would you accept a PR that adds a `--namespace` argument to your existing [`compilejsi18n`](http://django-statici18n.readthedocs.io/en/latest/commands.html#compilejsi18n) management command?  Any gotchas or contribution guidelines that we should be careful of?

Thanks!

**Comment by pomegranited** [comment]
Addressed by https://github.com/zyegfryed/django-statici18n/pull/41

**Comment by zyegfryed** [comment]
Fixed in v1.8.0. Thanks!
